upgrade/upgrade-scripts/upgrade

```diff
@@ -128,14 +128,26 @@ function upgrade_in_place() {
 	#
 	[[ "$DLPX_UPGRADE_DRY_RUN" == "true" ]] && return
 
+	CURRENT_VERSION=$(get_current_version) || die "failed to get version"
+
+	source_version_information
+
 	set_upgrade_property "UPGRADE_TYPE" "$UPGRADE_TYPE" ||
 		die "failed to set upgrade property 'UPGRADE_TYPE' to '$UPGRADE_TYPE'"
 
+	set_upgrade_property "UPGRADE_VERSION" "$VERSION" ||
+		die "failed to set upgrade property 'UPGRADE_VERSION' to '$VERSION'"
+
+	if [[ -n "$HOTFIX" ]]; then
```

Comment:
Where does this `HOTFIX` variable get set?

Reply:
IIRC, it comes from the `version.info` file in the upgrade image.. and it might not be set if it's not an image for a hotfix.. e.g. if you look at a regular upgrade image, that value will be `HOTFIX=`.. i.e. empty.. but for a hotfix it'll be `HOTFIX=HF-1234`..

Reply:
this is similar to the `VERSION` field, which also comes from the same file..